src/factories/Container.ts

```diff
@@ -0,0 +1,93 @@
+module n3Charts.Factory {
+  'use strict';
+
+  export interface IDimension {
+    width: number;
+    height: number;
+    innerWidth: number;
```

Comment:
good idea, my thoughts were that they are needed quite often - so it might be handy to have them directly on the dimension object (but of course generated automatically).